test/clj/game_test/cards/resources.clj

```diff
@@ -803,6 +803,22 @@
       (prompt-choice :runner "Card from hand")
       (prompt-choice :runner "Steal"))))
 
+(deftest gang-sign-correct-prompts
+         ;; Gang Sign - accessing from HQ, not including root. Issue #2113.
```

Comment:
This comment is too far indented. Should be aligned with `do-game` below.

Reply:
Also, you should move this to `deftest gang-sign` with `testing "correct prompts"` or similar. See Noah's great tests 😉